Management/src/ActuatorApi/appsettings.json

```diff
@@ -0,0 +1,46 @@
+{
+  // Steeltoe: Add schema to get auto-completion.
+  "$schema": "https://steeltoe.io/schema/latest/schema.json",
+  "Logging": {
+    "LogLevel": {
+      "Default": "Information",
+      "Microsoft.AspNetCore": "Warning",
+      "Steeltoe.Samples.ActuatorApi.Models.WeatherForecast": "Debug"
```

Comment:
I don't understand why this was added. There's no logging in `WeatherForecast`.

Reply:
I don't recall.... Debug doesn't belong in this file, I'll remove it